db/migrate/20220922113844_create_users.rb

```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :posts_counter
+      t.integer :postsCounter
       t.has_many :post
       t.has_many :comment
       t.has_many :like
```

Comment:
- [ ] for your app to run correctly, kindly remove this relationship, you have already done it in the models. and kindly remove any relationship you have in your other migration files, I saw many in your other migration files :+1: